contrib/mmdarwin/mmdarwin.c

```diff
@@ -407,16 +407,19 @@ CODESTARTnewActInst
 		}  else if (!strcmp(actpblk.descr[i].name, "response")) {
 			char *response = es_str2cstr(pvals[i].val.d.estr, NULL);
 
-			if (!strcmp(response, "no") != 0) {
+			if (!(strcmp(response, "no") != 0)) {
```

Comment:
I think these boolean expressions need clarification.

I suggest to replace them by either
`strcmp(resonse, "no") == 0`
or the well-knwon
`strcmp(resonse, "no")` - I think this should have been it in the first place,.

Reply:
Thank you for improving the readability of the code.